src/main/java/stsjorbsmod/cards/wanderer/Rediscovery.java

```diff
@@ -0,0 +1,43 @@
+package stsjorbsmod.cards.wanderer;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Wanderer;
+import stsjorbsmod.powers.RediscoveryPower;
+
+import static stsjorbsmod.JorbsMod.makeCardPath;
+
+public class Rediscovery extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Rediscovery.class.getSimpleName());
+    public static final String IMG = makeCardPath("Scaling_Rares/rediscovery.png");
+
+    private static final CardRarity RARITY = CardRarity.RARE;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.POWER;
+    public static final CardColor COLOR = Wanderer.Enums.WANDERER_CARD_COLOR;
+
+    private static final int COST = 3;
```

Comment:
I think the sheet says the intended design is "always 1 cost, upgrade to innate"